sdk/servicebus/service-bus/README.md

```diff
@@ -85,90 +71,100 @@ The following sections provide code snippets that cover some of the common tasks
 
 ### Send messages
 
-Once you have created an instance of a `QueueClient` or `SubscriptionClient` class, create a sender
-using the [createSender](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/queueclient#createsender--)
-function. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-sendablemessageinfo-) messages.
+Once you have created an instance of a `ServiceBusClient` class, you can get a `Sender`
+using the [getSender](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient#getsender-string-)
+function. This gives you a sender which you can use to [send](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/sender#send-servicebusmessage-)
+messages.
```

Comment:
Breaking in between the lines?
(Same at many other places.)

Reply:
These represent my struggle between wanting to follow some sort of reasonable "make this look nice if you're just viewing the markdown and not a rendering of the markdown". I can remove it but that's why I keep doing it :)

Reply:
Fixed - it actually made a nice sentence on it's own so it was win-win.